ftw/tabbedview/browser/resources/tabbedview.js

```diff
@@ -483,21 +489,19 @@ load_tabbedview = function(callback) {
     tabbedview.reload_view();
   });
 
-  // dynamic batching functionality
-  $('#tabbedview-batchbox').live('blur', function() {
-    $('#dynamic_batching_form').submit();
-  });
-
-  // workaround to bind the submit event with live
-  $('body').each(function(){
-    $('#dynamic_batching_form').live('submit', function(event){
-      tabbedview.param(
-        'pagesize', $('#tabbedview-batchbox').val());
-      tabbedview.flush_params('pagenumber:int');
-      tabbedview.reload_view();
-      event.preventDefault();
-    });
-  });
+  $('.tabbedviewBatchbox').live(
```

Comment:
`live` is deprecated use `on` instead:
```javascript
$(document)
  .on('blur', '.tabbedviewBatchbox', () => { /*...*/ })
  .on('keypress', '.tabbedviewBatchbox', () => { /*...*/ })
```

Reply:
`live` was used before for this event and is used everywhere in this file (local consistency). 

It's an old file and I only want to change what I really need to.

Updating `live` to `on` should be done within a new maintenance-issue and should not be part of this PR.